docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -125,6 +132,56 @@ show up in ``DESCRIBE <table-name>`` or ``SELECT *``.
 
 This property is optional; the default is ``true``.
 
+``kafka.security-protocol``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To enable ``SSL`` for the Kafka connection, set this property to ``SSL``.
+
+This property is optional; the default ist ``PLAINTEXT``.
+
+``kafka.keystore-location``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Location of the keystore file used for connection to Kafka cluster.
+
+This property is optional.
+
+``kafka.keystore-password``
```

Comment:
Why do you use ` `` ` instead of `?

Reply:
This was from the previous committer's PR. I'll change that for a single quote **`**

Reply:
Looking at the previous Kafka configs, it seems that double backticks `` were already being used. Would you prefer modifying all of them for single backtick ` ?